sdk/core/azure-core-test/src/test/java/com/azure/core/test/implementation/RestProxyWithMockTests.java

```diff
@@ -166,14 +161,6 @@ public void service1GetDateTimeRfc1123() {
         assertEquals(OffsetDateTime.ofInstant(Instant.ofEpochMilli(0), ZoneOffset.UTC), dateTime);
     }
 
-    @Test
```

Comment:
Tests not required anymore?

Reply:
I can add this test back, it was just relying on an illegal access to implementation in Azure Core since it was configured as Java 8.

Reply:
I've seen quite a few of these illegal access warnings. Is there a plan within the Java team to track them down and eliminate them? I'm assuming we don't want to disable all the tests where they occur, and instead just need to find another route to get the same test coverage?